scripts/jarvis.js

```diff
@@ -163,3 +161,77 @@ Everything is running 🎊
     return true;
   });
 }
+
+const informAboutInitialSetup = () => {
+  const rlp = require('readline');
```

Comment:
Is that a Node API? because if it's an npm package it will crash because the user hasn't run the npm install project yet.

Reply:
yes it is https://www.geeksforgeeks.org/node-js-readline-module/

Reply:
Oh nice didn't know that !